Fix #1470 - Left margin added to text-view

areebmomin opened the pull request:
Fixes #1470 

![issue_after](https://user-images.githubusercontent.com/43715712/81847208-23231480-9508-11ea-8142-4c3c577ba5a0.png)


Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

areebmomin committed:
docs: Change some confusing words

areebmomin committed:
change BaseURL

areebmomin committed:
Merge remote-tracking branch 'upstream/development' into development
new update

ShivangiSingh17 reviewed (changes requested):
Can you remove the previously committed changes as they are not part of this PR. Also, remove the base change URL file as its already been solved in another PR.

areebmomin committed:
Update README.md

areebmomin commented:
@ShivangiSingh17  I have just reverted the changes that I have made in readme.md file instead of removing the commit. The reason behind is that i have changed two files under a single commit.
